### PR TITLE
The Webservice key must have a length of 32 characters

### PR DESCRIPTION
**Pull request by wgroenewold**
Key length is checked to be 32 chars long so hinting that would be nice.

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Made example code more specific to prevent errors.
| Fixed ticket? | No

### CODE REVIEW

File: src/content/1.7/webservice/tutorials/creating-access.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ Creating API keys can be done with the class WebserviceKey.
 ```php
 <?php
 $apiAccess = new WebserviceKey();
-$apiAccess->key = 'GENERATE_A_COMPLEX_VALUE';
+$apiAccess->key = 'GENERATE_A_COMPLEX_VALUE_32_CHAR';
```
Comment (PierreRambaud) [inline src/content/1.7/webservice/tutorials/creating-access.md:68]:
```suggestion
$apiAccess->key = 'GENERATE_A_COMPLEX_VALUE_WITH_32_CHARACTERS';
```

**Comment by matthieu-rolland** [comment]
thank you @wgroenewold !

File: src/content/1.7/webservice/tutorials/creating-access.md (hunk 1 above)
Comment (wgroenewold) [reply src/content/1.7/webservice/tutorials/creating-access.md]: @PierreRambaud Now your example still isn't 32 chars but 43 chars.

Comment (wgroenewold) [reply src/content/1.7/webservice/tutorials/creating-access.md]: ```32_CHARACTERS_COMPLEX_VALUE_HERE``` 

would be a better example imho.

**Comment by wgroenewold** [comment]
@matthieu-rolland @PierreRambaud ```32_CHARACTERS_COMPLEX_VALUE_HERE ``` would be a better example since it is 32 characters long instead of @PierreRambaud his 43 characters long example. That would make a working example.

**Comment by PierreRambaud** [comment]
> @matthieu-rolland @PierreRambaud `32_CHARACTERS_COMPLEX_VALUE_HERE ` would be a better example since it is 32 characters long instead of @PierreRambaud his 43 characters long example. That would make a working example.

Having a wrong value always force people to use the correct value because the app will not work without it :)

**Pull request merged**